admin/test/test_release.py

```diff
@@ -1264,23 +1337,26 @@ def test_marketing_repositories_created(self):
             build_server=self.create_fake_repository(files=repo_contents),
         )
 
-        expected_files = set()
-        for operating_system in self.operating_systems:
-            for file in [
-                'clusterhq-flocker-cli-0.3.3-1.noarch.rpm',
-                'clusterhq-flocker-node-0.3.3-1.noarch.rpm',
-                'repodata/repomd.xml',
-            ]:
-                path = os.path.join(
-                    operating_system['distro'],
-                    operating_system['version'],
-                    operating_system['arch'],
-                    file,
-                )
-                expected_files.add(path)
+        expected_files = [
```

Comment:
I changed these tests because I thought it would be better to have consistency between similar tests and the way they were done was horrible.